pkg/controllers/vrpq/verticarestorepointsquery_controller.go

```diff
@@ -102,9 +102,11 @@ func (r *VerticaRestorePointsQueryReconciler) SetupWithManager(mgr ctrl.Manager)
 // constructActors will a list of actors that should be run for the reconcile.
 // Order matters in that some actors depend on the successeful execution of
 // earlier ones.
-func (r *VerticaRestorePointsQueryReconciler) constructActors(_ *vapi.VerticaRestorePointsQuery,
-	_ logr.Logger) []controllers.ReconcileActor {
+func constructActors(r *VerticaRestorePointsQueryReconciler, vrpq *vapi.VerticaRestorePointsQuery,
```

Comment:
Why did you move this function out of the `VerticaRestorePointsQueryReconciler` struct?

Reply:
oops, Will move it back